keadm/cmd/keadm/app/cmd/util/cloudinstaller.go

```diff
@@ -112,13 +113,23 @@ func (cu *KubeCloudInstTool) TearDown() error {
 	cu.SetKubeEdgeVersion(cu.ToolVersion)
 
 	// Kill cloudcore process
-	if err := cu.KillKubeEdgeBinary(KubeCloudBinaryName); err != nil {
-		return err
+	if cu.HelmInstallMode {
+		// For cloudcore container install, we need to run `helm uninstall cloudcore -n kubeedge` command and clean kubeedge namespace
+		uninstall := NewCommand("helm uninstall cloudcore -n kubeedge")
```

Comment:
I think we had better use the helm API to uninstall it. Users will not install helm.  I will pr keadm init-beta later.